metadata-ingestion/tests/unit/stateful_ingestion/state/test_checkpoint.py

```diff
@@ -128,3 +128,20 @@ def test_serde_idempotence(state_obj):
         config_class=MySQLConfig,
     )
     assert orig_checkpoint_obj == serde_checkpoint_obj
+
+
+def test_supported_encodings():
```

Comment:
should we support reading state serialized with utf-8 but allow writing it back with base85?

Reply:
Actually it's already supported. Here is workflow with stateful ingestion, based on kafka ingester:
1. User calls `get_last_checkpoint`, which reads checkpoint aspect and decode it's payload based on `serde` field. Default encoding before this change was utf-8.
2. User creates new checkpoint, writes data in it. Default encoding now is base85
3. User commits checkpoint. If user does not explicitly change `serde` of new checkpoint to utf-8, then it would be written with base85. So, after ingestion checkpoint would be overwritten with base85 encoding

Reply:
Yes, @alexey-kravtsov is spot on.